numpy/_pytesttester.py

```diff
@@ -23,6 +23,8 @@
 In practice, tests run from the numpy repo are run in develop mode. That
 includes the standard ``python runtests.py`` invocation.
 
+This module is imported by every numpy subpackage, so lies at the top level to
+simplify circular import issues.
```

Comment:
Maybe add something like "This is also why numpy and pytest are not imported on top, but rather inside `_show_numpy_info` and `PytestTester.__call__`"